app/src/openxr/cpp/OpenXRInputSource.h

```diff
@@ -72,6 +72,7 @@ class OpenXRInputSource {
     bool mHasAimState { false };
     XrPosef mHandAimPose;
     bool mSupportsFBHandTrackingAim { false };
+    double mSmoothIndexThumbDistance { 0 };
```

Comment:
maybe this could jsut be an static variable inside the lambda as it does not seem part of the object status...